src/win32/fs.cpp

```diff
@@ -1476,9 +1476,9 @@ WinDirNotify::~WinDirNotify()
 #endif
 }
 
-FileAccess* WinFileSystemAccess::newfileaccess(bool followSymLinks)
+std::unique_ptr<FileAccess> WinFileSystemAccess::newfileaccess(bool followSymLinks)
 {
-    return new WinFileAccess(waiter);
+    return std::unique_ptr<FileAccess>(new WinFileAccess(waiter));
```

Comment:
Use {} to call constructors

Reply:
I prefer the original syntax, unless it's something that needs curly braces.  It's more backward compatible too.

Reply:
fair enough